apps/services/routers.py

```diff
@@ -262,6 +262,30 @@ async def read_request_details(
     }
 
 
+@services_router.get("/{university_id}/return_user_document/{user_request_id}",
+                     name="return_user_document",
```

Comment:
don't use return

Reply:
How should I rename it? Because I saw this "return" in Jira, so that's why I decided to call it this way 
Because we have another endpoint "read user documents"
`Get user documents list`?